### DISCUSSION TITLE
What happened to Integral? Why isn't the Integral movement a bigger deal than it is? (Or it is just "everywhere" and so invisible)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#937] [General]
For several years now I've been asking people questions like:

> What happened to Integral?
> 
> Why isn't the Integral movement a bigger deal than it is?
>
> Is Integral a bit "passé? And if so, why? (e.g. few of the active people i know actively self-describe as integral and there doesn't seem to be *that* active an integral community)

This came out of noticing that ... 

- Integral was often not talked about or referenced explicitly (even where clearly an influence)
- People seemed to think that integral not amount to as much as it promised - cf https://markmanson.net/ken-wilber 
- Ken Wilber (and Integral) sometimes treated a bit like the “Dirty uncle in the attic”[^1] i.e. someone important but who everyone seems a bit ashamed about

Why ... (reasons I've heard from people or read)

- Lack of footnoting and rigor in the work - over-playing his hand intellectually
- Ken being a bit crazy/difficult/... (?)
- "Branding" of integral as a community (e.g. as old white guys)
- Integral being activating for "green": academically, socially …
- Integral movement being a bit “in-group” 
- Integral institute "imploding"

Personal asides:

- Comparing Karl Marx and Ken wilber 😉 (Marx wasn't much read or appreciated in his time! Das Kapital barely sold any copies and Marx died b4 finishing the 2nd volume)
- NB: i'm asking this because i'm genuinely confused. Integral doesn't seem to get as much attention as it should do ... and few of the people i encounter describe themselves as card-carrying integralists (even if many have read integral works)

[^1]: This was a phrase said to me by someone when talking about this question.

**Comment by matthewmccarthy11** [comment]
From my own experience I can say that I never came across 'integral' in any of my college courses, but I think that people now in college generally would be responsive to integral, if they were exposed to it.

**Reply by rufuspollock** [reply]
@matthewmccarthy11 i think the issue is that integral theory is a "grand theory of everything" that academics (for some good reasons) are very wary of - plus Wilber's approach to footnoting and evidence is definitely not up to academic standards. I think quite a few academics are aware of Integral or even influenced by it but very few would say that or explicitly reference e.g. Wilber's work.

**Comment by matthewmccarthy11** [comment]
Following up here on the university point: what kinds of university courses/programs/universities might be more open to teaching and using integral theory? Should there be a 'course' on integral if so what would that look like?

**Comment by lorenzsell** [comment]
When you say integral are you primarily referring to Ken Wilbur's work?  I often wonder about whether when people say Integral they ever also include Sri Aurobindo's approach to Integral Yoga, which far precedes Wilbur's work.

**Reply by rufuspollock** [reply]
@lorenzsell yes i mean Integral as primarily referring to what grew up around the work of Ken Wilber and others (there are a variety of people there but Wilber is the best known).

NB: the term integral as used by him does indeed come (AFAIK) from Aurobindo via Gebser. And Wilber explicitly references Aurobindo.
